sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/LogsIngestionClient.java

```diff
@@ -96,7 +127,15 @@ public void upload(String ruleId, String streamName, Iterable<Object> logs) {
     /**
      * Uploads logs to Azure Monitor with specified data collection rule id and stream name. The input logs may be
      * too large to be sent as a single request to the Azure Monitor service. In such cases, this method will split
-     * the input logs into multiple smaller requests before sending to the service.
+     * the input logs into multiple smaller requests before sending to the service. This method will block until all
+     * the logs are uploaded or an error occurs. If an {@link LogsUploadOptions#setLogsUploadErrorConsumer(Consumer) error handler} is set,
```

Comment:
```suggestion
     * the logs are uploaded or an error occurs. If a {@link LogsUploadOptions#setLogsUploadErrorConsumer(Consumer) error handler} is set,
```

Reply:
For the user, this reads as `If an error handler is set`. They will not read the type and method name inside `{}`.